### PR TITLE
reload multipath configuration file

### PR DESCRIPTION
**Pull request by Promaethius**
This solves https://github.com/packethost/packet-block-storage/issues/9 by reloading the multipath configuration file. Allows the use of this script for CoreOS.

### CODE REVIEW

**Comment by nathangoulding** [comment]
@dlaube Can you have a look over this?

**Comment by nathangoulding** [comment]
@Promaethius can you address @dlaube's comments around conditionally doing this for CoreOS hosts to address the problem of errors on other platforms?

**Comment by Promaethius** [comment]
Added a section that will check for dist.

File: README.md [hunk 1]
```diff
@@ -35,8 +35,8 @@ This utility/helper script is made available to assist in  the attachment and de
 **Using wget**
 
     sudo su -
-    wget -O /usr/bin/packet-block-storage-attach https://raw.githubusercontent.com/packethost/packet-block-storage/master/packet-block-storage-attach
-    wget -O /usr/bin/packet-block-storage-detach https://raw.githubusercontent.com/packethost/packet-block-storage/master/packet-block-storage-detach
+    wget -O /usr/bin/packet-block-storage-attach https://raw.githubusercontent.com/promaethius/packet-block-storage/master/packet-block-storage-attach
+    wget -O /usr/bin/packet-block-storage-detach https://raw.githubusercontent.com/promaethius/packet-block-storage/master/packet-block-storage-detach
```
Comment (nathangoulding) [inline README.md]: Can you revert this so we can get this merged in? Thanks!

**Comment by Promaethius** [comment]
Done.

**Pull request closed without merging**